_api-reference/profile.md

```diff
@@ -753,4 +753,283 @@ Field | Description
 `post_collection`| Contains the time spent running the aggregation’s `postCollection()` callback method.
 `build_aggregation`| Contains the time spent running the aggregation’s `buildAggregations()` method, which builds the results of this aggregation.
 `reduce`| Contains the time spent in the `reduce` phase.
-`<method>_count` | Contains the number of invocations of a `<method>`. For example, `build_leaf_collector_count` contains the number of invocations of the `build_leaf_collector` method. 
\ No newline at end of file
+`<method>_count` | Contains the number of invocations of a `<method>`. For example, `build_leaf_collector_count` contains the number of invocations of the `build_leaf_collector` method. 
+
+## Concurrent segment search
+
+Starting with OpenSearch 2.10, you can perform [concurrent segment search]({{site.url}}{{site.baseurl}}/search-plugins/concurrent-segment-search/), in which each shard-level request will search the segments in parallel during query phase. If you enable concurrent segment search, the Profile API response will contain several additional fields with statistics about _slices_.
+
+A slice is the unit of work that can be executed by a thread. Each query can be partitioned into multiple slices, with each slice containing one or more segments. All the slices can be executed either in parallel or in some order depending on the available threads in the pool.
+
+In general, the max/min/avg slice time captures statistics across all slices for a timing type. For example, when profiling aggregations, the `max_slice_time_in_nanos` field in the `aggregations` section shows the maximum time consumed by the aggregation operation and its children across all slices. 
+
+#### Example response
+
+The following is an example response for a concurrent search with three segment slices:
```

Comment:
Should we mention this response has profile enabled?

Reply:
This file is the Profile API documentation and it states in the very beginning that you have to set `profile` to `true` in order to get these responses.

Reply:
@ticheng-aws are any of these response fields gated by the concurrent search feature flag? If so I think we should call those out.

Reply:
The new fields are not gated by the concurrent search feature flag.